lib/ansible/modules/storage/glusterfs/gluster_volume.py

```diff
@@ -153,6 +153,16 @@
       - 192.0.2.10
       - 192.0.2.11
   run_once: true
+
+- name: Remove the bricks from gluster volume
+  gluster_volume:
+    state: present
+    name: testvol
+    bricks: /bricks/brick1/b1,/bricks/brick2/b2
+    cluster:
+      - 10.70.42.85
+    force: true
+  run_once: true
```

Comment:
Add an example for removing bricks in case of replica as well. With replica option.

Reply:
@sac Added an example to reduce cluster configuration.